source/index.html.md

```diff
@@ -104,6 +104,153 @@ curl -X DELETE \
 
 You can access Kontist API endpoints by adding `Authorization`: `Bearer AUTH_TOKEN` header to your requests.
 
+## Device Binding
+
+To have access to Kontist API endpoints that require strong customer authentication, you need to pass two-factor authentication (2FA).
+
+To make it seamless for you we provide device binding authentication that leverages digital signature algorithm.
+
+To pass device binding authentication you need to generate a pair of private and public keys by using elliptic curve algorithm  (secp256r1). To create and verify your device you need to pass your public key and then sign OTP from SMS received on your phone.
+
+After that, you can use your device id and your private key to get your confirmed auth token by creating and verifying device challenge.
+
+### Create device
+
+To initiate the device binding you need to create a device. After a successful request, you will receive an SMS with OTP that you will need during device verification.
+
+```shell
+curl "https://api.kontist.com/api/user/devices" \
+  -H "Authorization: Basic QWxhZGRpbjpPcGVuU2VzYW1l" \
```

Comment:
Do we need to use Basic Auth here? I think we should use our unconfirmed JWT

Reply:
answered below